server/mdm/apple/cert.go

```diff
@@ -160,17 +157,3 @@ func NewDEPKeyPairPEM() ([]byte, []byte, error) {
 
 	return publicKeyPEM, privateKeyPEM, nil
 }
-
-func DecryptBase64CMS(p7Base64 string, cert *x509.Certificate, key crypto.PrivateKey) ([]byte, error) {
```

Comment:
Just moved this out of Apple-specific mdm package to a common mdm one as it is shared between Apple and Windows.